### PR TITLE
GH-1153 Adds new PropEdit

### PR DESCRIPTION
**Pull request by GordonSmith**
Fix leaking watches.
Fix Shape Colours.
Added "render=false" to prevent contained widgets or widgetArrays from auto rendering.

Fixes GH-1153

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage (rebased back to 1.10.0 branch) @mlzummo, @dtsnell4 Please Review

**Comment by dtsnell4** [comment]
Looks good to me

File: src/other/PropertyEditor.js [hunk 1]
```diff
@@ -1,401 +1,406 @@
...
-                        .paramGrouping( selectedItems.length > 1 ? "By Param" : "By Widget")
-                        .render();
-                };
-            }
+            this.watchWidget(_);
         }
         return retVal;
     };
-
-    PropertyEditor.prototype.show_settings = function (_) {
```
Comment (mlzummo) [inline src/other/PropertyEditor.js]: this is goign to cause problems in DSP

Comment (GordonSmith) [inline src/other/PropertyEditor.js]: @mlzummo:  show_settings re-added

**Comment by GordonSmith** [comment]
@mlzummo Please re-review...

**Comment by jbrundage** [comment]
Looks good to me

**Pull request merged**